src/screens/Services/Services.js

```diff
@@ -49,14 +48,12 @@ const mastercardIcon = require('assets/icons/mastercard.png');
 class ServicesScreen extends React.Component<Props> {
   getServices = () => {
     const {
-      navigation, theme, offers, baseFiatCurrency,
+      navigation, theme, baseFiatCurrency, providersMeta,
     } = this.props;
     const colors = getThemeColors(theme);
 
-    const providersCount = uniq((offers || []).map(offer => offer.provider)).length;
-
-    const offersBadge = providersCount ? {
-      label: `${providersCount} exchanges`,
+    const offersBadge = Array.isArray(providersMeta) ? {
```

Comment:
Hm, maybe we would need to check if providersMeta.length > 0? Otherwise we might end up showing "0 exchanges" if something goes wrong. Maybe then it's better not to show anything?

Reply:
great catch!